test/lightning/impact_tracking/worker_test.exs

```diff
@@ -1,12 +1,15 @@
 defmodule Lightning.ImpactTracking.WorkerTest do
-  use ExUnit.Case, async: false
+  use Lightning.DataCase, async: false
 
-  alias Lightning.ImpactTracking.{Client, Worker}
+  alias Lightning.ImpactTracking.{Client, Configuration, Report, Worker}
 
   import Mock
   import Lightning.ApplicationHelpers, only: [put_temporary_env: 3]
 
   @host "https://foo.bar"
+  @metrics %{}
```

Comment:
what you think about testing to save a non empty map? would be cool to see some example close to actual metrics.

Reply:
Watch this space :) - the next Lightning PR will contain the actual metrics.